src/app/content/practiceQuestions/components/PQButton.tsx

```diff
@@ -1,33 +1,19 @@
-import styled, { css } from 'styled-components/macro';
-import { PlainButton } from '../../../components/Button';
-import { linkColor, textRegularSize } from '../../../components/Typography';
-import theme from '../../../theme';
-
-const sharedStyles = css`
-  background-color: ${theme.color.primary.orange.base};
-  color: ${theme.color.text.white};
-  ${textRegularSize}
-  font-weight: bold;
-  height: 5rem;
-  padding: 0 3rem;
-`;
-
-// tslint:disable-next-line: variable-name
-const PQButton = styled(PlainButton)`
-  ${sharedStyles}
-`;
+import styled from 'styled-components/macro';
+import Button from '../../../components/Button';
+import { linkColor } from '../../../components/Typography';
 
 // tslint:disable-next-line: variable-name
-export const PQInput = styled.input`
-  ${sharedStyles}
-  border: none;
-  cursor: pointer;
+const PQButton = styled(Button)`
   ${(props: { disabled: boolean }) => props.disabled && `
     background-color: #f1f1f1;
     color: #c1c1c1;
     cursor: not-allowed;
+    &:hover {
+      background-color: #f1f1f1;
+    }
   `}
   ${(props: { withoutBg?: boolean }) => props.withoutBg && `
```

Comment:
you could add this as a variant to the button component. call it `variant="transparent"` or `variant="dropout"` or something